DepthImages/obstacle_detection.py

```diff
@@ -31,20 +33,20 @@ def detect(input, width, threshold):
 
     while index < len(array) - width:
         # Slope calculation
-        x1 = index
         y1 = array[index]
-        x2 = index + width
         y2 = array[index + width]
 
-        slope = (y2 - y1) / (x2 - x1)
+        slope = (y2 - y1) / width
 
-        index += width
+        index += width	# why are we doing this? it skips <width> points, is it just to be faster? operating under the assumption that any points between those we're comparing would yield similar results?
```

Comment:
Your theory might be correct but I'm not too sure for this one, might be best to ask Sharare

Reply:
Can you add her to our repository?